lib/emojis.dart

```diff
@@ -36,123 +14,514 @@ class AnimatedEmojis {
+    thumbsDown,
+    thumbsDownLight,
+    thumbsDownMediumLight,
+    thumbsDownMedium,
+    thumbsDownMediumDark,
+    thumbsDownDark,
+    raisingHands,
+    raisingHandsLight,
+    raisingHandsMediumLight,
+    raisingHandsMedium,
+    raisingHandsMediumDark,
+    raisingHandsDark,
+    wave,
+    waveLight,
+    waveMediumLight,
+    waveMedium,
+    waveMediumDark,
+    waveDark,
+    victory,
+    victoryLight,
+    victoryMediumLight,
+    victoryMedium,
+    victoryMediumDark,
+    victoryDark,
+    crossedFingers,
+    crossedFingersLight,
+    crossedFingersMediumLight,
+    crossedFingersMedium,
+    crossedFingersMediumDark,
+    crossedFingersDark,
+    indexFinger,
+    indexFingerLight,
+    indexFingerMediumLight,
+    indexFingerMedium,
+    indexFingerMediumDark,
+    indexFingerDark,
+    foldedHands,
+    foldedHandsLight,
+    foldedHandsMediumLight,
+    foldedHandsMedium,
+    foldedHandsMediumDark,
+    foldedHandsDark,
+    dancerWoman,
+    dancerWomanLight,
+    dancerWomanMediumLight,
+    dancerWomanMedium,
+    dancerWomanMediumDark,
+    dancerWomanDark,
     rose,
+    wiltedFlower,
+    fallenLeaf,
     plant,
     luck,
+    snowflake,
+    volcano,
+    sunrise,
+    sunriseOverMountains,
     rainbow,
+    windFace,
+    electricity,
     dizzy,
+    comet,
+    globeShowingEuropeAfrica,
+    unicorn,
+    lizard,
+    dragon,
+    tRex,
     turtle,
+    snake,
+    frog,
+    rabbit,
+    rat,
+    dog,
+    pig,
+    racehorse,
+    donkey,
+    ox,
+    goat,
+    kangaroo,
+    tiger,
+    monkey,
+    chipmunk,
+    otter,
+    bat,
+    rooster,
+    hatchingChick,
+    babyChick,
+    hatchedChick,
+    eagle,
+    peace,
+    goose,
+    peacock,
+    seal,
+    dolphin,
+    whale,
+    blowfish,
     crab,
     octopus,
+    jellyfish,
+    snail,
+    ant,
+    mosquito,
+    bee,
     butterfly,
+    pawPrints,
+    tomato,
+    popcorn,
+    hotBeverage,
     clinkingBeerMugs,
+    clinkingGlasses,
     bottleWithPoppingCork,
+    wineGlass,
+    tropicalDrink,
     policeCarLight,
+    flyingSaucer,
     rocket,
+    airplaneDeparture,
+    airplaneArrival,
+    rollerCoaster,
+    confettiBall,
     balloon,
     birthdayCake,
     fireworks,
+    mirrorBall,
     soccerBall,
+    directHit,
+    violin,
+    drum,
+    maracas,
+    batteryFull,
+    batteryLow,
     moneyWithWings,
+    lightBulb,
+    graduationCap,
+    umbrella,
     gemStone,
+    alarmClock,
     bellhopBell,
     bell,
+    aries,
+    taurus,
+    gemini,
+    cancer,
+    leo,
+    virgo,
+    libra,
+    scorpio,
+    sagittarius,
+    capricorn,
+    aquarius,
+    pisces,
+    ophiuchus,
+    exclamationDouble,
+    crossMark,
     musicalNotes,
-    cool
+    checkMark,
+    cool,
+    plusSign,
+    chequeredFlag,
   ];
 
+  /// Returns the name of the emoji from the [id] in camel case.
+  ///
+  /// For example: `u1f603` => smileWithBigEyes
+  static String getCamelCaseName(String id) {
+    const idToNames = {
```

Comment:
This list is missing some ids. For example, when I'm trying to display unicorn emoji, I get a null check error.